src/modules/mavlink/mavlink_high_latency2.cpp

```diff
@@ -203,10 +203,6 @@ bool MavlinkStreamHighLatency2::write_airspeed(mavlink_high_latency2_t *msg)
 	airspeed_s airspeed;
 
 	if (_airspeed_sub.update(&airspeed)) {
-		if (airspeed.confidence < 0.95f) { // the same threshold as for the commander
-			msg->failure_flags |= HL_FAILURE_FLAG_DIFFERENTIAL_PRESSURE;
-		}
-
```

Comment:
@acfloria as I think you were the one that implemented this: any problem with removing the check for the low confidence here? If yes, then I propose we should subscribe to `airspeed_validated` here, as there you would get notified if the airspeed got invalid (it will set it to NAN). 
Plus, was the airspeed confidence level check working for your setups? Were you able to e.g. detect stuck sensors?

Reply:
If you removed now the confidence and replace it with airspeed_validated it would be nice to also reflect that here.

We never had a faulty airspeed sensor during flights so I cannot report if the confidence was working properly.

Reply:
@acfloria like this? https://github.com/PX4/Firmware/pull/14254/commits/324f4fd6150639b6a3c797be2b910a6ec024f89a I'm a HITL novice, so don't know what the goal is that we want to achieve with it exactly. Is the check there just to indicate it if you do not have an airspeed sensor connected to your HITL setup, as part of the preflight checks?

Reply:
Yeah, this seems to be correct. So this message was not used in HIL but when flying with a satellite communication link. Since this message is only sent while flying we did not use it for the preflight checks but as an indication if the airspeed sensor would have been faulty during flight. I think the way you implemented it now it would exactly serve that purpose again.

Reply:
Ah of course, now it makes more sense^^ Don't know where I got the idea from that it was HIL stuff.. Okay, thanks for the mini-review!